src/hotspot/cpu/aarch64/vm_version_aarch64.cpp

```diff
@@ -250,6 +250,11 @@ void VM_Version::initialize() {
     FLAG_SET_DEFAULT(UseVectorizedMismatchIntrinsic, false);
   }
 
+#ifdef __ARM_FEATURE_ATOMICS
+// Compile time LSE support is enabled
+// Forcibly enable runtime LSE support
+  FLAG_SET_DEFAULT(UseLSE, true);
+#else
```

Comment:
I don't believe this block does anything useful, and it disables `-XX:-UseLSE` for testing.

Reply:
Agree, I restored original detection code (it just does what we need on the right hardware).